website/blog/2024-10-23-release-0.76-new-architecture.md

```diff
@@ -0,0 +1,224 @@
+
+## Breaking Changes
+
+### Removed the dependency on the @react-native-community/cli
+
+React Native should be [framework agnostic](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0759-react-native-frameworks.md). We completed the work to remove the `@react-native-community/cli` as a direct dependency of React Native.
+
+Decoupling React Native from the CLI allows us to move faster through releasing these projects independently.
+
+If you are relying on the CLI in your daily workflow, make sure to explicitly [add the dependency](https://react-native-community.github.io/upgrade-helper/?from=0.75.4&to=0.76.0#RnDiffApp-package.json) on the CLI in your `package.json`:
+
+```diff
+//…
+  “devDependencies”: {
+   // …
++  “@react-native-community/cli”: “15.0.0”,
++  "@react-native-community/cli-platform-android": “15.0.0”,
++  "@react-native-community/cli-platform-ios": “15.0.0”,
+  },
+```
+
+### [Android] React Native is distributed as a single library
+
+React Native 0.76 will ship with a reduced number of native dynamic libraries, as we merged a lot of our native code into a single library called `libreactnative.so`.
+This change comes with reduction in app size, and improvement in app startup performance on Android.
+
+On the other hand, this is a breaking change for both app and library developers.
+
+App developers will have to update their Application’s `onCreate` as follows:
+
+```diff
++import com.facebook.react.soloader.OpenSourceMergedSoMapping
+import com.facebook.soloader.SoLoader
+
+class MainApplication : Application(), ReactApplication {
+
+  override fun onCreate() {
+    super.onCreate()
++   SoLoader.init(this, OpenSourceMergedSoMapping)
+-   SoLoader.init(this, false)
+  }
+}
+```
+
+This change is necessary in order to properly load `libreactnative.so` and is included in the upgrade-helper.
+
+Library authors won’t be affected by this change, unless you have custom C++ code.
+For a technical deep dive into this change, and suggestions for library authors, you can read more about it [in this dedicated post](https://github.com/react-native-community/discussions-and-proposals/discussions/816).
+
+### Updates to Minimum iOS and Android SDK requirements
+
+We have updated our minimum platform and SDK versions:
+
+- iOS: [15.1](https://support.apple.com/en-gb/108051#151)
+- Android: [SDK 24](https://developer.android.com/tools/releases/platforms#7.0) (Android 7)
+
+This change was announced earlier in the year when 0.75 [was released](/blog/2024/08/12/release-0.75#last-version-supporting-minsdk-23-and-miniosversion-134). For more background, please see the dedicated posts for [Android](https://github.com/react-native-community/discussions-and-proposals/discussions/802) and [iOS](https://github.com/react-native-community/discussions-and-proposals/discussions/812).
+
+### Other Breaking Changes
+
+- **Animation**
```

Comment:
Why is this capitalized and the other bullets are lower case? Should they all be capitalized?

Reply:
We capitalized proper modules (`Animation`, `Pressable`, ...) and we left generic concepts lowercase ("infra", "runtime").
We can change it, if we prefer for it to be all capital case.